packages/elements/src/calendar/index.ts

```diff
@@ -83,11 +84,12 @@ import type {
   WeekdayName
 } from './types';
 
-export { CalendarFilter };
+export type { CalendarFilter, BeforeCellRenderEvent };
```

Comment:
`CalendarCell` should be re-exported again at `index.ts`
```suggestion
export type { CalendarCell, CalendarFilter, BeforeCellRenderEvent };
```